queries/haskell/rainbow-parens.scm

```diff
@@ -0,0 +1,60 @@
+(exp_apply (_
+  (("(" @opening)
+   (")" @closing)))) @container
```

Comment:
Why are you matching `exp_apply` instead of `exp_parens`? With `exp_parens` you do not need the extra `_` in your pattern.

```suggestion
(exp_parens 
  (("(" @opening)
   (")" @closing))) @container
```

Reply:
Not sure why I had chosen that....